doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -160,6 +160,17 @@ Zigbee
 
 |no_changes_yet_note|
 
+Gazell
+------
+
+* Added:
+
+   * User-specific file for Gazell pairing. User can override Gazell pair config inside application.
```

Comment:
```suggestion
   * User-specific file for Gazell pairing.
     User can override Gazell pair config inside the application.
```

Reply:
Consider referring to Kconfig option names to simplify looking for newly added options e.g.:

```suggestion
   * :kconfig:option:`CONFIG_GAZELL_PAIRING_CONFIG_USER` and :kconfig:option:`CONFIG_GAZELL_PAIRING_CONFIG_FILE_NAME`.
      The options allow to use user-specific file as Gazell pairing configuration header to override the pairing configuration.
```